pom.xml

```diff
@@ -382,41 +382,44 @@
         <groupId>org.revapi</groupId>
         <artifactId>revapi-maven-plugin</artifactId>
         <configuration>
+          <failBuildOnProblemsFound>false</failBuildOnProblemsFound>
+          <pipelineConfiguration>
+            <transformBlocks>
+              <block>
+                <item>manually-vetted</item>
+                <item>revapi.versions</item>
+              </block>
+            </transformBlocks>
+          </pipelineConfiguration>
```

Comment:
Isn't this part of the parent pom? You can check by calling `mvn help:effective-pom`.